engine/src/main/java/org/terasology/engine/modes/StateIngame.java

```diff
@@ -117,6 +119,9 @@ public Boolean get() {
 
     @Override
     public void dispose(boolean shuttingDown) {
+        ScreenGrabber screenGrabber = context.get(ScreenGrabber.class);
+        screenGrabber.takeGamePreview(PathManager.getInstance().getSavePath(gameManifest.getTitle()));
```

Comment:
Just to clarify, do we need to then garbage collect the actual `screenGrabber` instance?

Reply:
To be honest, I just took a look at the decompiled class and compiler already simplified this as one complex line. Maybe better to leave it as is. I believe it's just could make code more readable.